tests/conftest.py

```diff
@@ -431,15 +431,15 @@ def site_alert():
 
 @pytest.fixture(autouse=True)
 def rap_api_token(settings):
-    """Django settting for the token for communicating with the RAP API."""
+    """Django setting for the token for communicating with the RAP API."""
```

Comment:
I really like this small "quality of life" improvement. It's easy to skip over mistakes like this, but making these small changes add up over time.